bee-runtime/src/resource.rs

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use log::warn;
+
+use std::{
+    any::{type_name, Any},
+    collections::HashMap,
+    ops::Deref,
+    panic::Location,
+    sync::{
+        atomic::{AtomicUsize, Ordering},
+        Arc, Mutex, Weak,
+    },
+};
+
+static RES_ID: AtomicUsize = AtomicUsize::new(0);
```

Comment:
```suggestion
static RESOURCE_ID: AtomicUsize = AtomicUsize::new(0);
```

Reply:
This is an implementation detail. The type is itself already named `ResHandle` so I think it's fairly clear as to its meaning.

Reply:
Changed.